### PR TITLE
Update meeting template

### PR DESCRIPTION
**Pull request by patcon**

### CODE REVIEW

File: -meeting-template.md [hunk 1]
```diff
@@ -1,22 +1,42 @@
...
-[📑 Retreat Planning Board](https://github.com/cryptographydog/december-meetup/projects/1)  
+<sup>[from template][template]</sup>
 
-Time:   
-Location: https://appear.in/offline  
-Attending:  
+[:date: Calendar][cal] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive]
 
-[TOC]
+Time: ...
```
Comment (dcwalk) [inline -meeting-template.md]: Can we add two trailing spaces so the line break is rendered in MD?

Comment (dcwalk) [reply -meeting-template.md]: Same for the two lines below!

Comment (dcwalk) [inline -meeting-template.md]:
```suggestion
Time: ...  
```

File: -meeting-template.md [hunk 2]
```diff
@@ -1,22 +1,42 @@
...
+Location: https://appear.in/offline
```
Comment (dcwalk) [inline -meeting-template.md]:
```suggestion
Location: https://appear.in/offline  
```

File: -meeting-template.md (hunk 1 above)
Comment (patcon) [reply -meeting-template.md]: oh! i had to idea about that! I've been removing them like a doofus!

File: -meeting-template.md [hunk 3]
```diff
@@ -1,22 +1,42 @@
...
 
-[🗓 Calendar](https://calendar.google.com/calendar/embed?src=s2224p8sptnujs736vplf9anjo%40group.calendar.google.com&ctz=America%2FToronto)  
-[📑 Retreat Planning Board](https://github.com/cryptographydog/december-meetup/projects/1)  
+<sup>[from template][template]</sup>
 
-Time:   
+[:date: Calendar][cal] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive]
+
+Time: ...  
 Location: https://appear.in/offline
```
Comment (benhylau) [inline -meeting-template.md]:
```suggestion
Location: https://meet.jit.si/offline  
```

**Pull request merged**